src/specular/stages/batches.rs

```diff
@@ -76,7 +82,7 @@ where
     fn try_next(&mut self) -> Result<Option<Batch>> {
         let batcher_transaction = self.batcher_transaction_iter.next();
         if let Some(batcher_transaction) = batcher_transaction {
-            let batches = decode_batches(&batcher_transaction, &self.state, self.config.chain.blocktime)?;
+            let batches = decode_batches(&batcher_transaction, &self.state, &self.config)?;
```

Comment:
Is passing the full config `self.config` the right way? Can we limit it to just the needed data?

Reply:
rust generally makes this more annoying to do than golang